module/component/component_manager_test.go

```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/onflow/flow-go/utils/unittest"
 )
 
-const CHANNEL_CLOSE_LATENCY_ALLOWANCE = 20 * time.Millisecond
+const CHANNEL_CLOSE_LATENCY_ALLOWANCE = 25 * time.Millisecond
```

Comment:
Should we add the test reproducing the race? It's `TestComponentManagerShutdown` in https://github.com/onflow/flow-go/commit/803b173a753dffe658c9c0695f59cff5a1f2d893